src/data/content/design-system/fondamenti/icone.yaml

```diff
@@ -0,0 +1,286 @@
+            icon:
+              icon: "sprites.svg#it-flag"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Progettare e sviluppare l'interfaccia utente #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Progettazione interfaccia, Sviluppo interfaccia]
+  
+  sectionsEditorial:
+    - full: true
+      noSpace: false
+      paddingLeft: true
+      title: null
+      text: null
+      background: null
+      components:
+
+        - name: TextImageCta
+          title: A cosa servono
+          headingLevel: 2
+          text: |
+            Le icone sono elementi visivi che contengono simboli: sono cruciali per un'interfaccia di qualità e hanno lo scopo di veicolare informazioni agli utenti in modo rapido e immediato. 
+            
+            Possono rappresentare **funzioni**, **categorie**, **opzioni** e molto altro, migliorando l'interazione in termini di usabilità e accessibilità. Ciò permette agli utenti di navigare e interagire con l'applicazione o il sito web in modo più intuitivo. 
+            
+            Le icone devono essere facilmente riconoscibili e comprensibili, garantendo un design coerente nell'ambito della soluzione digitale. 
+        
+        - name: TextImageCta
+          title: Come usarle
+          headingLevel: 2
+          text: |
+            Usa le icone solo quando è strettamente necessario e per i seguenti casi d'uso:
+
+            - **Rappresentazione di funzioni**: puoi usare le icone per simboleggiare funzioni specifiche dell'interazione come "salva", "cancella" o "invia". È importante usarle in modo consistente nelle varie ricorrenze per evitare di confondere gli utenti.
+            - **Navigazione**: le icone possono segnalare percorsi di navigazione, come ad esempio l'icona del menu a hamburger. I simboli delle icone devono essere intuitivi per rendere la navigazione più semplice.
+            - **Categorie di contenuto**: le icone possono rappresentare categorie di contenuti, come "video", "immagini" o "documenti". I simboli devono essere facilmente riconoscibili e in linea con il contenuto rappresentato.
+            - **Feedback visivo**: puoi usare le icone per fornire feedback visivo, come ad esempio un'icona di caricamento o di successo. È fondamentale usarle nel modo corretto per comunicare chiaramente lo stato dell'azione dell'utente.
+            - **Interazioni**: le icone possono rappresentare interazioni quali "mi piace", "condividi" o "commenta". In questo caso i simboli devono essere facilmente comprensibili e azionabili per agevolare l'interazione desiderata dall'utente.
+
+            Ti consigliamo di abbinare un'icona a un'etichetta di testo che ne chiarisce la funzione. Puoi tuttavia usare le sole icone se il simbolo che contengono indica chiaramente e senza ambiguità l'azione possibile.
```

Comment:
```suggestion
            Ti consigliamo di abbinare un'icona a un'etichetta di testo o a un tooltip (suggerimento contestuale) che ne chiarisce la funzione. Questo consente di spiegare l'azione senza ambiguità. Puoi tuttavia usare le sole icone se il simbolo che contengono indica chiaramente l'azione possibile.
```

Reply:
Renderei meno ambigua anche l'ultima frase. Qualcosa che spieghi che sì, puoi usare le sole icone nell'interfaccia, ovviamente però se metti in campo tecniche perché chi naviga con ad es. Lettori di schermo abbia comunque un'etichetta da leggere, o un testo alternativo a disposizione, per capirne la funzione. Forse basta una virgola alla fine e qualcosa integrativo.

Reply:
Ho rivisto l'ultima frase, cercando di essere conciso ma chiaro.

Reply:
@danielaiozzo e @Fupete appena potete, date uno sguardo all'ultima integrazione per capire se ci siamo :)